isabelle/Doc/Specification/document/root.tex

```diff
@@ -59,6 +59,7 @@
 Hernán Rajchert \and \\
 Brian Bush
 }
+\date{2 Dec 2022}
```

Comment:
The date on the PDF is "December 31, 1979".

Reply:
The attached pdf has 2 of December. Are you building it local?

Reply:
Yes, I'm building locally, according to the instructions in the repo.